controllers/dataprotection/backup_controller.go

```diff
@@ -132,7 +146,8 @@ func (r *BackupReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		WithOptions(controller.Options{
 			MaxConcurrentReconciles: viper.GetInt(maxConcurDataProtectionReconKey),
 		}).
-		Owns(&batchv1.Job{})
+		Owns(&batchv1.Job{}).
```

Comment:
need own statefulset?

Reply:
instead of watching the pod of StatefulSet.  beacuse sometimes the sts will no any changes, if pod is always crashloopBackof.